docs/configure-and-manage/manage/json-rpc-api-security.md

```diff
@@ -55,6 +55,17 @@ let provider = new HttpHeaderProvider('https://...', headers);
 web3.setProvider(provider);
 ```
 
+### `web3j-quorum`
+
+Use the [web3j-quorum](https://github.com/web3j/web3j-quorum) Java library:
+
+```java
+HttpService service = new HttpService("https://...");
```

Comment:
```suggestion
HttpService service = new HttpService("<JSON-RPC HTTP endpoint>");
```

Reply:
We generally place replaceable code between the "<>" characters. Not sure if my description is correct though

Reply:
Want to nudge people to wards using https when using auth